pyvista/core/dataset.py

```diff
@@ -2568,3 +2575,61 @@ def cell_point_ids(self, ind: int) -> List[int]:
         cell = self.GetCell(ind)
         point_ids = cell.GetPointIds()
         return [point_ids.GetId(i) for i in range(point_ids.GetNumberOfIds())]
+
+    def point_is_inside_cell(
+        self, ind: int, point: Union[VectorArray, NumericArray]
+    ) -> Union[int, np.ndarray]:
+        """Return whether a point is inside a cell.
+
+        .. versionadded:: 0.35.0
+
+        Parameters
+        ----------
+        ind : int
+            Cell ID.
+
+        point : Sequence(float) or np.ndarray
```

Comment:
I don't really know the specification of these fields, but how standard is `Sequence(float)`? I can only see this in `_coerce_pointslike_arg()` and `find_containing_cell()`, `find_closest_cell()`. If it's like `typing.Sequence` then perhaps it should be `Sequence[float]`?

OK, we have both in https://dev.pyvista.org/api/core/_autosummary/pyvista.DataSet.find_closest_cell.html?highlight=find_closest_cell#pyvista.DataSet.find_closest_cell vs https://dev.pyvista.org/api/core/_autosummary/pyvista.core.common_data.perlin_noise.html?highlight=perlin_noise, and neither get picked up by intersphinx so it probably doesn't really matter either way.

Reply:
Went with square brackets.